### PR TITLE
Fixed Hyperlink in BrayCurtis Distance

### PR DESCRIPTION
**Pull request by krutikabapat**
Updated conf.py.in and braycurtis.rst to reflect changes in the hyperlink of the **BrayCurtis Distance** in the reference  section of 
 https://www.shogun-toolbox.org/examples/latest/examples/distance/braycurtis.html

### CODE REVIEW

**Comment by krutikabapat** [comment]
@vigsterkr Is there any further change required in this?

File: doc/cookbook/conf.py.in [hunk 1]
```diff
@@ -315,6 +315,7 @@ texinfo_documents = [
 extlinks = {
 'sgissue': ('https://github.com/shogun-toolbox/shogun/issues/%s',        'issue '),
 'sgclass': ('http://www.shogun-toolbox.org/%s',        ''),
-'wiki': ('https://en.wikipedia.org/wiki/%s',        'Wikipedia: ')
+'wiki': ('https://en.wikipedia.org/wiki/%s',        'Wikipedia: '),
+'BrayCurtis_Distance':('https://people.revoledu.com/kardi/tutorial/Similarity/%s.html','BrayCurtis_Distance')
```
Comment (karlnapf) [inline doc/cookbook/conf.py.in:319]: this is not something we want to do.

File: doc/cookbook/source/examples/distance/braycurtis.rst [hunk 2]
```diff
@@ -33,7 +33,7 @@ We can use the same instance with new :sgclass:`DenseFeatures` to compute asymme
 ----------
 References
 ----------
-'BrayCurtis Distance <http://people.revoledu.com/kardi/tutorial/Similarity/BrayCurtisDistance.html>'
+:BrayCurtis_Distance:`BrayCurtisDistance`
```
Comment (karlnapf) [inline doc/cookbook/source/examples/distance/braycurtis.rst:36]: I am confused why you did this? Was there any problem?

**Comment by gf712** [comment]
@karlnapf shall I close this then?

**Pull request closed without merging**